contrib/core/actions/send_mail/send_mail

```diff
@@ -8,9 +8,12 @@ TO=$1
 shift
 SUBJECT=$1
 shift
+SEND_EMPTY_BODY=$1
+shift
 BODY="$@"
 
-${MAIL} <<EOF
+if [[ -z "${BODY// }" && $SEND_EMPTY_BODY == 'True' ]] || [[ -n $BODY ]]; then
```

Comment:
That's a good question. I assumed that when a parameter is of type "boolean", there are some other internal checks of the myriad of "true" and "false" values and ultimately set the parameter to True or False, being the Python bool values. That's all a huge assumption and I was hoping someone would flag it, like has just been done :smile: